### DISCUSSION TITLE
Logging and alerting on realtime-js connection subscription error/disconnects.

### DISCUSSION THREAD

**Discussion by kevinco26** [supabase/supabase#18343] [General]
Hi, I posted a question on stack overflow here. https://stackoverflow.com/questions/77326772/how-to-set-multiple-clients-and-channels-in-supabase-realtime

Essentially I'm looking for documentation on two things:
1. How many connections of websockets could I open against multiple supabase client in a nodejs application (is it dependent on the vm I'm running the app?)
2. It's easy to subscribe to a topic. What happens if that connection gets dropped/closed/etc. I would like to know when this happens so that I can handle appropriately with my own alerting system.. but I don't know how to get notified if the current websocket connection that I created with the supabase sdk went down or not

**Comment by GaryAustin1** [comment]
I use something like this to detect errors in subscription.

```
            .subscribe((status) => {
                console.log('subscribe_status, ' + status);
                if (status !== 'SUBSCRIBED') {
                    connectionErrorHandler(status)
                }
            })
```
